pkg/scheduler/apis/config/v1beta2/default_plugins.go

```diff
@@ -147,19 +147,34 @@ func mergePlugins(defaultPlugins, customPlugins *v1beta2.Plugins) *v1beta2.Plugi
 	return defaultPlugins
 }
 
+type pluginIndex struct {
+	index  int
+	plugin v1beta2.Plugin
+}
+
 func mergePluginSet(defaultPluginSet, customPluginSet v1beta2.PluginSet) v1beta2.PluginSet {
 	disabledPlugins := sets.NewString()
+	enabledCustomPlugins := make(map[string]pluginIndex)
 	for _, disabledPlugin := range customPluginSet.Disabled {
 		disabledPlugins.Insert(disabledPlugin.Name)
 	}
-
+	for index, enabledPlugin := range customPluginSet.Enabled {
+		enabledCustomPlugins[enabledPlugin.Name] = pluginIndex{index, enabledPlugin}
```

Comment:
only the first repetition?

Reply:
true, seems like we'd better to validate this config to avoid the case of repetition, will fix it in my next day!

Reply:
recall that code freeze is on Thursday

Reply:
because `mergePlugins` is done before the validation, so the repetition here is still possible, I'd suggest to just log something here or revert to the original version, only the last repetition will be valid for either way.
I am a little lost here, do you have any suggestion?

Reply:
You can keep it like this, it doesn't really matter because more than one repetition will fail later.

Fix the types.go though